tracer/test/Datadog.Trace.Tests/Configuration/IntegrationSettingsTests.cs

```diff
@@ -27,6 +30,24 @@ public void IntegrationEnabled(string settingName, string settingValue, bool exp
             Assert.Equal(expected, settings.Enabled);
         }
 
+        [Theory]
+        [InlineData("false", "true", true)]
+        [InlineData("true", null, true)]
+        [InlineData(null, "false", false)]
+        public void CaseInsenstiveIntegrationEnabled(string sensitiveCaseValue, string upperCasedValue, bool expected)
```

Comment:
I know it's only on test case, but I think it can be improved before merging this PR. The main motivation of the PR is to test the casing of the environment variable name, so can you refactor the test so that the only data being applied to it is the name of the environment variable and/or the result? From my understanding, we want to test that:
- `DD_TRACE_MySql_ENABLED=false` (as before) disables this integration that's on by default
- `DD_TRACE_MYSQL_ENABLED=false` (all caps, new behaviour) disables this integration that's on by default

Reply:
Updated in: https://github.com/DataDog/dd-trace-dotnet/pull/6175/commits/51d99733d20cecf41731ab9f95575819608b70f0